package.json

```diff
@@ -13,13 +13,15 @@
     "rollup": "^2.3.4",
     "rollup-plugin-css-only": "^3.1.0",
     "rollup-plugin-livereload": "^2.0.0",
-    "rollup-plugin-svelte": "^7.0.0",
+    "rollup-plugin-postcss": "^4.0.2",
+    "rollup-plugin-svelte": "^7.1.0",
     "rollup-plugin-terser": "^7.0.0",
-    "svelte": "^3.46.2"
+    "svelte": "^3.59.2"
   },
   "dependencies": {
+    "@sveltestrap/sveltestrap": "^6.2.7",
     "sirv-cli": "^2.0.0",
     "svelte-step-wizard": "0.0.2",
-    "sveltestrap": "^5.7.0"
+    "sveltestrap": "^5.11.3"
```

Comment:
Already to version 5.11! Unbelievable!

Reply:
Yes, I wanted to add tailwind too but I couldn't.